sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -324,6 +324,12 @@ async def __anext__(self):
             _LOGGER.warning("Incomplete download: %s", err)
             internal_response.close()
             raise IncompleteReadError(err, error=err)
+        except aiohttp.client_exceptions.ClientResponseError as err:
+            raise ServiceResponseError(err, error=err) from err
+        except aiohttp.client_exceptions.ClientError as err:
+            raise ServiceRequestError(err, error=err) from err
+        except asyncio.TimeoutError as err:
```

Comment:
small change to swap the position of `asyncio.TimeoutError` with `ClientError` as the latter is the base class for all aiohttp errors.